editor/src/components/common/shared-strategies/convert-to-flex-strategy.spec.browser2.tsx

```diff
@@ -572,6 +572,129 @@ describe('Smart Convert To Flex', () => {
 	`),
     )
   })
+
+  it('converts groups with constraints that imply a certain row-like layout', async () => {
+    const editor = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(
+        `
+			<div style={{ ...props.style }} data-uid='a'>
+        <Group
+          data-uid='parent'
+          style={{
+            height: 204,
+            position: 'absolute',
+            left: 14,
+            top: 343,
+            width: 584,
+          }}
+        >
+          <span
+            data-uid='second'
+            style={{
+              wordBreak: 'break-word',
+              fontSize: '31px',
+              height: 119,
+              position: 'absolute',
+              left: 91,
+              top: 29,
+              width: 'max-content',
+            }}
+            data-constraints={['left']}
+          >
+            Hello World
+          </span>
+          <div
+            data-uid='third'
+            style={{
+              backgroundColor: '#FF69B4AB',
+              height: 204,
+              contain: 'layout',
+              width: 119,
+              position: 'absolute',
+              left: 465,
+              top: 0,
+            }}
+            data-constraints={['right', 'width']}
+          />
+          <div
+            data-uid='first'
+            style={{
+              backgroundColor: '#FF69B4AB',
+              height: 204,
+              contain: 'layout',
+              width: 59,
+              position: 'absolute',
+              left: 0,
+              top: 0,
+            }}
+            data-constraints={['left', 'width']}
+          />
+        </Group>
+			</div>
+	  `,
+      ),
+
+      'await-first-dom-report',
+    )
+
+    await convertParentToFlex(editor)
+
+    expect(getPrintedUiJsCode(editor.getEditorState())).toEqual(
+      makeTestProjectCodeWithSnippet(`
+		<div style={{ ...props.style }} data-uid='a'>
+      <Group
```

Comment:
looks like there is a missing Group -> div conversion step! (probably related that the data-constraints are left on the children too)

Reply:
Ah, I wasn't sure if we wanted this because it came up previously. But it certainly makes sense in this narrow case to make that change.

Reply:
Should be fixed by e08982bb4b318b31fe5221f15fdbc35691bd20eb.